src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,35 @@
 package core.basesyntax.service;
 
+import core.basesyntax.CustomException;
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int VALID_AGE = 18;
+    private static final int MAX_VALID_AGE = 115;
+    private static final int VALID_COUNT_OF_CHARACTER = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws CustomException {
+        if (user == null) {
+            throw new CustomException("User is null");
+        }
+        if (user.getAge() < VALID_AGE || user.getAge() > MAX_VALID_AGE || user.getAge() == null) {
+            throw new CustomException("Age should be above 18 or equal and less than 115!");
```

Comment:
You can add to your message constant. VALID_AGE, MAX_VALID_AGE. It better if in future you decide change limits, you won't need to change message.
And you can add ", but was" + user.getAge()    to you message. For more informative